### PR TITLE
#161545951 Create template for PR document

### PR DESCRIPTION
**Pull request by caveinn**
## What does this PR do?
- Include the creation of a PR template document

## Description of Task to be completed?
- Create a template that can be used to quickly create a pull request.
- Add a fix for the travis badge.

#### How should this be manually tested
N/A

#### Any background context you want to provide?
- When you raise a pull request you will be provided with a template for editing to fit your need.

#### What are the relevant pivotal tracker stories?

[#161545951](https://www.pivotaltracker.com/n/projects/2198533/stories/161545951)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Comment by lukorito** [comment]
@caveinn This pr needs to rebase

**Comment by caveinn** [comment]
@lukorito I have rebased. You happened to comment immediately after another pr was merged.

**Comment by lukorito** [comment]
> @lukorito I have rebased. You happened to comment immediately after another pr was merged.

There was a PR merged today?

**Review by michael-basweti — COMMENTED** [review]
This is good Nzioka but here is something you may want to look at. You can add a screenshot of a PR example so that one can refer if they don't have an idea on how to use the template

**Comment by caveinn** [comment]
@michael-basweti I have included a link to a sample pr instead since a screenshot would not cover the entire pr.

**Review by michael-basweti — APPROVED** [review]
Nice work, The changes have been addressed

**Review by IncredibleBlackMan — APPROVED** [review]
This looks good to me.

**Comment by Arthur236** [comment]
Hey @caveinn. Change the titles from `####` to `##` and add the below sections before the `## Screenshots`

```
## Packages installed
None

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos
- [x] Raise PR
- [ ] Test
- [ ] Documentation

```

**Comment by caveinn** [comment]
> Hey @caveinn. Change the titles from `####` to `##` and add the below sections before the `## Screenshots`
> 
> ```
> ## Packages installed
> None
> 
> ## Deployment note
> N/A
> 
> ## Related PRs branch | PR (branch_name) | (pr_link)
> Branch | PR
> ------ | ------
> other_pr_production | [link]()
> other_pr_master | [link]()
> 
> ## Todos
> - [x] Raise PR
> - [ ] Test
> - [ ] Documentation
> ```

I have made the changes you have requested

File: PR_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## How should this be manually tested?
+
+## Any background context you want to provide?
+
+## What are the relevant pivotal tracker stories?
+
+[# ](link)
+
+## Screenshots (if appropriate)
```
Comment (Arthur236) [inline PR_TEMPLATE.md]: Move this below the `## Todos`

File: PR_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+------ | ------
+other_pr_production | [link]()
+other_pr_master | [link]()
+
+## Todos
+- [x] Raise PR
+- [ ] Test
+- [ ] Documentation
+
+
```
Comment (Arthur236) [inline PR_TEMPLATE.md]: Remove the extra line

**Comment by Arthur236** [comment]
@caveinn 
Update your PR description. Also, by having the `PULL_REQUEST_TEMPLATE.md` file in the `.github/` directory, you would not need to copy the contents to your PR as you are creating it since it will come pre-populated with the contents in the template.

**Comment by caveinn** [comment]
> @caveinn
> Update your PR description. Also, by having the `PULL_REQUEST_TEMPLATE.md` file in the `.github/` directory, you would not need to copy the contents to your PR as you are creating it since it will come pre-populated with the contents in the template.

I have implemented those changes.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Description of Task to be completed?
+
+## How should this be manually tested?
+
+## Any background context you want to provide?
+
+## What are the relevant pivotal tracker stories?
+
+
```
Comment (Arthur236) [inline .github/PULL_REQUEST_TEMPLATE.md]: One final nitpick. Remove this extra line

**Pull request merged**